content/compatibility/python.adoc

```diff
@@ -0,0 +1,15 @@
+= Artificial Intelligence planning optimization in Python
+:jbake-type: compatibilityBase
+:jbake-description: Use OptaPy (open source) for Artificial Intelligence planning optimization on Java.
+:jbake-priority: 1.0
+:jbake-related_tag: python
+:showtitle:
+
+https://www.optapy.org[OptaPy] allows you to run OptaPlanner from Python code,
+at a performance cost.
```

Comment:
My main issue with "at a performance cost" is that it currently a very high performance cost, to the point "at a performance cost" doesn't really state how high the performance cost is (for example, the benchmarker/monitoring is at a performance cost, but nowhere near the performance cost of OptaPy)

Reply:
```suggestion
at a high performance cost.
```